evm/p2p/test_peer.py

```diff
@@ -174,3 +198,7 @@ class MockP2PProtocol(P2PProtocol):
 
     def send_handshake(self):
         pass
+
+
+class MockChain:
```

Comment:
This doesn't appear to be used anywhere.

Reply:
Indeed, removed